### ISSUE TITLE
Release with cuDNN 4

### ISSUE THREAD

**Issue by bashimao** [bytedeco/javacpp-presets#201]
I realize that there is a open issue where people request cuDNN 5 to be implemented. However, cuDNN 4 is already out since February. The code is already there, but we have not had a production release since last fall. Can you create an maven-enabled artifact that contains the modifications for cuDNN 4? That would really help me a lot.

**Comment by saudet** [comment]
Updating cuDNN is going to be trivial work compared to what I need to do for a full release. But I do plan to make one before long, so let's hope for the best :) cuDNN 5 would be alright, right? No need of cuDNN 4 in particular?

**Comment by bashimao** [comment]
Well.. In particular I want to use batch normalization, which was introduced in cuDNN 4. However, this technique has caught so much interest it will undoubtedly also be present in cuDNN 5. So yes, I am fine with either. The major problem that I have is that my supervisor doesn't want me to build own artifacts of third party stuff if possible because we do not want to allocate manpower to maintain such custom builds. It is a little bit sad, since I can see from the commit-log that cuDNN 4 support has already been updated in February this year. So tempting... But... well... Would it be possible to do a minor release (1.1a or 1.1.1) that just contains the current CUDA updates?

Is there any way that I could help?

**Comment by saudet** [comment]
Help would be great, yes :) Do you need that to run under CentOS 6? I'm thinking of upgrading the builds to CentOS 7 because TensorFlow builds only with great difficulty on CentOS 6... If you need CentOS 6, could you figure out the procedure with Docker to make TensorFlow build? Here's what I'm doing now but it's not enough:
https://github.com/bytedeco/javacpp-presets/wiki/Build-Environments#linux-x86-and-x86_64
That would help a lot.

**Comment by saudet** [comment]
Version 1.2 has been released! Thank you for your patience
